components/Ui/UiCompactFeature.vue

```diff
@@ -15,6 +18,7 @@ interface Props {
   description: string;
   icon: string;
   title: string;
+  detail: string;
```

Comment:
if you have a `v-if` on L10 checking for `detail`, should this line be:

```suggestion
  detail?: string;
```
??